components/tab/src/tab/tab.js

```diff
@@ -72,9 +72,11 @@ const Tab = ({
                 text-overflow: ellipsis;
                 transition: fill 150ms ease-in-out;
             }
-
-            button:focus span {
-                outline: 1px dotted ${colors.grey900};
+            /*todo: decide below*/
+            /*avoids the focus-after-click that was reported as confusing. not supported by safari though, possible to polyfill?*/
+            button:focus-visible {
```

Comment:
Using `:focus-visible` here is nice because it avoids the "focus-on/after-click" that has been reported as confusing. It's not yet out of [CSS4 draft](https://caniuse.com/css-focus-visible), though it has decent coverage. Is this a bad idea?

Reply:
According to https://css-tricks.com/platform-news-using-focus-visible-bbcs-new-typeface-declarative-shadow-doms-a11y-and-placeholders/, a good way to deal with lack of widespread coverage is to define focus styles in `:focus` and then undo those in `:focus-visible`.

Reply:
Nice! Thanks @mediremi. That seems like a reasonable workaround to me -- I can make the changes necessary.